From 55bd14c586f0f64b8ee5515161be1aa1098d3226 Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Tue, 3 Jul 2012 22:28:05 -0400
Subject: [initscripts][PATCH] remove /run/nologin before leaving rc.multi
To: arch-projects@archlinux.org

systemd-186 writes this file when running systemd-tmpfiles, but
initscripts never removes it. This disallows all non-root logins.

Reported-by: Dan McGee <dan@archlinux.org>
Signed-off-by: Dave Reisner <dreisner@archlinux.org>
---
 rc.multi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rc.multi b/rc.multi
index daf2372..7d4acaa 100755
--- a/rc.multi
+++ b/rc.multi
@@ -31,4 +31,6 @@ run_hook multi_end

 bootlogd_stop

+rm -f /run/nologin
+
 # vim: set ts=2 sw=2 noet:
--
1.7.11.1
